pages/platform/ai/notebook_tuto_10_create_chatbot/guide.fr-fr.md

```diff
@@ -0,0 +1,127 @@
+---
+title: AI Notebooks - Tutoriel - Créer et entraîner un chatbot Rasa
```

Comment:
@Victor2103 , @eleapttn , please confirm the FR title and excerpt are OK, or suggest alternatives.